src/app/body/setup/setup.component.ts

```diff
@@ -187,5 +187,35 @@ export class SetupComponent implements OnInit {
     } catch (error) {
       console.log(error);
     }
+
+    try {
+      const result = await callable({ mode: "patchModerator", Patch: "Patch4", PatchName: "Patch-4", PatchDescription: "This patch allows the user to change a particular field in relevent tasks, enter field name and field value to get the task details", CreationDate: "07/07/2021", UpdatedOn: "12/08/2021", LastUsedByOrg: "", LastUsedByUid: ""}).toPromise();
+
+      console.log("Created Patch3 document");
```

Comment:
It should be patch4 here and the description is the same as patch3. I think the mode is different for patch4.